WalletWasabi/Gma/QrCodeNet/Encoding/EncodingRegion/Codeword.cs

```diff
@@ -54,7 +54,7 @@ internal static void TryEmbedCodewords(this TriStateMatrix tsMatrix, BitList cod
 
 		if (bitIndex != codewordsSize)
 		{
-			throw new Exception($"Not all bits from {nameof(codewords)} consumed by matrix: {bitIndex} / {codewordsSize}.");
+			throw new ArgumentException($"Not all bits from {nameof(codewords)} consumed by matrix: {bitIndex} / {codewordsSize}.");
```

Comment:
I think we should just configure Deepsource to ignore code libraries like this QrCode.Net library, and we should not change their code.